packages/cli/src/deploy/getWorldProxyFactoryContracts.ts

```diff
@@ -0,0 +1,96 @@
+import accessManagementSystemBuild from "@latticexyz/world/out/AccessManagementSystem.sol/AccessManagementSystem.json" assert { type: "json" };
+import balanceTransferSystemBuild from "@latticexyz/world/out/BalanceTransferSystem.sol/BalanceTransferSystem.json" assert { type: "json" };
+import batchCallSystemBuild from "@latticexyz/world/out/BatchCallSystem.sol/BatchCallSystem.json" assert { type: "json" };
+import registrationSystemBuild from "@latticexyz/world/out/RegistrationSystem.sol/RegistrationSystem.json" assert { type: "json" };
+import initModuleBuild from "@latticexyz/world/out/InitModule.sol/InitModule.json" assert { type: "json" };
+import initModuleAbi from "@latticexyz/world/out/InitModule.sol/InitModule.abi.json" assert { type: "json" };
+import worldProxyFactoryBuild from "@latticexyz/world/out/WorldProxyFactory.sol/WorldProxyFactory.json" assert { type: "json" };
+import worldProxyFactoryAbi from "@latticexyz/world/out/WorldProxyFactory.sol/WorldProxyFactory.abi.json" assert { type: "json" };
+import { Hex, getCreate2Address, encodeDeployData, size } from "viem";
+import { salt } from "./common";
+import { Contract } from "./ensureContract";
+
+export function getWorldProxyFactoryContracts(deployerAddress: Hex): Record<string, Contract & { address: Hex }> {
```

Comment:
would it make sense to have this call `getWorldFactoryContracts`? or pull out the shared behavior into something like `getWorldDependencies`?

Reply:
also any chance we could pull out this refactor into a separate PR (along with `getDeployer`)?

Reply:
Lets do that! it essential works for now

Reply:
done!